dev/com.ibm.websphere.appserver.features/visibility/public/messagingClient-3.0/resources/l10n/io.openliberty.messagingClient-2.0.properties

```diff
@@ -0,0 +1,22 @@
+###############################################################################
+# Copyright (c) 2020 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+#
+#ISMESSAGEFILE FALSE
+#NLS_ENCODING=UNICODE
+#NLS_MESSAGEFORMAT_NONE
+#
+
+description=The messagingClient-3.0 feature provides applications with access to the message \
+ queues that are hosted on WebSphere Application Server through the Jakarta Messaging 3.0 API. \
+ The messagingClient-3.0 feature is compliant with Jakarta Messaging 3.0 specifications and is \
+ supported only on JDK 8 or later. It enables access to the messaging engine that is enabled \
```

Comment:
IBM style specifies "Integration Bus" should be capitalized

Reply:
I'm finding that SIB terms are either all capitalized or all lower case.  All are now capitalized.